src/hooks/useCanInteractWithColony.ts

```diff
@@ -48,10 +48,24 @@ export const useCanInteractWithColony = (colony?: Colony): boolean => {
   if (!wallet || !colony) {
     return false;
   }
+
+  /*
+   * Check if connected to the same chain
+   */
   const [{ id: walletHexChainId }] = wallet.chains;
   const colonyChain = colony?.meta?.chainId || DEFAULT_NETWORK_INFO.chainId;
   const userWalletChain = parseInt(walletHexChainId.slice(2), 16);
-  return colonyChain === userWalletChain && canInteractWithNetwork;
+
+  /*
```

Comment:
I felt this check was relevant here as it is also relevant to whether can interact with a Colony or not.